system-tests/test/video_compression_spec.js

```diff
@@ -39,11 +39,12 @@ describe('e2e video compression', () => {
   ].forEach((headed) => {
     systemTests.it(`passes (head${headed ? 'ed' : 'less'})`, {
       // videos are corrupted in firefox due to known issues
-      browser: '!firefox',
+      browser: ['chrome', 'electron'],
```

Comment:
Why was this changed? Was webkit also causing issues?

Reply:
webkit was failing this test on my local machine with the error:

```
WebKit was launched, but the experimental feature was not enabled. Please add `experimentalWebKitSupport: true` to your config file to launch WebKit.
Error: WebKit was launched, but the experimental feature was not enabled. Please add `experimentalWebKitSupport: true` to your config file to launch WebKit.
```

So i assumed running the test against webkit was a waste 🤷‍♂️ 

Maybe it runs differently in CI?

Reply:
I see it being set [here](https://github.com/cypress-io/cypress/blob/c7da9f4ee61e3900af11a9e811934351ee5b0998/system-tests/lib/system-tests.ts#L934) but I haven't tried to run it locally. I assume it was working in CI before. Maybe just change it back?